packages/eslint-config-airbnb/test/test-base.js

```diff
@@ -1,13 +1,13 @@
 import test from 'tape';
 import base from '../base';
 
-test('base: does not reference react', t => {
-  t.plan(2);
+test('base: does not reference react', tst => {
```

Comment:
yeah - but in that case we should ammend the styleguide as well to say 1-letter vars are ok in some cases